src/git_repositories.ml

```diff
@@ -16,13 +16,15 @@ module Repositories = struct
 
     type t = {
       opam_repository_master : repo;
+      opam_repository_mingw_opam2 : repo;
```

Comment:
At some point later this year this will go - might it be better to have a list of repositories here?

Reply:
I think it doesn't matter much, if the patch is sufficiently clean it can simply be reverted, but maybe a list would help managing multiple repositories. Ok!

Reply:
Hm, no, it's simpler with a flat structure (no list), because the branch we want to checkout is different for each git repo (master for opam-repository, opam2 for opam-repository-mingw). With a list we won't be able to distinguish the repos, so we'd need to attach the branch to clone. It would induce a somewhat complex refactor, while reverting the commit is simple enough to remove the support for opam-repository-mingw.